subprojects/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/dsl/ModuleVersionMatcherParsersTest.groovy

```diff
@@ -25,7 +25,7 @@ import static org.gradle.api.internal.artifacts.DefaultModuleVersionSelector.new
 import static org.gradle.api.internal.artifacts.dsl.ModuleVersionSelectorParsers.multiParser
 import static org.gradle.api.internal.artifacts.dsl.ModuleVersionSelectorParsers.parser
 
-public class ModuleVersionSelectorParsersTest extends Specification {
+public class ModuleVersionMatcherParsersTest extends Specification {
```

Comment:
This should still say "selector", since the Strings we parse are called "selectors"